icu4c/source/config/dist.mk

```diff
@@ -25,15 +25,16 @@ DISTY_IN=$(DISTY_DATA)/in
 DOCZIP=icu-docs.zip
 
 SVNTOP=$(top_srcdir)/..
-SVNVER=$(shell svnversion $(SVNTOP) | cut -d: -f1 | tr -cd 'a-zA-Z0-9')
-SVNURL=$(shell svn info $(SVNTOP) | grep '^URL:' | cut -d: -f2-)
+GITVER=$(shell (cd $(SVNTOP) && git describe --tags --exact-match || git rev-parse --short HEAD) 2>/dev/null || echo 'unknown')
```

Comment:
If `cd $(SVNTOP)` fails then this command line will attempt to execute `git rev-parse` which I assume is wrong, one more set of parentheses would be needed in order to try first `git describe` then `git rev-parse` but only if `cd` succeeded.